ros_buildfarm/common.py

```diff
@@ -444,3 +444,14 @@ def topological_order_packages(packages):
         if pkg_path is None:
             raise RuntimeError('Circular dependency in: %s' % pkg)
     return ordered_pkg_tuples
+
+
+def get_node_label(
+    config_job_label, rosdistro_name=None, job_type=None, build_name=None
+):
+    if config_job_label is not None:
+        return config_job_label
+    label = 'buildslave'
+    if rosdistro_name and job_type and build_name:
```

Comment:
Why do you support all but not some of the optional arguments? It doesn't make much sense to me to return the default label if 2 of the 3 optional arguments are set.